cfgov/unprocessed/css/molecules/full-width-text.less

```diff
@@ -6,10 +6,15 @@
       markup: |
         <div class="o-full-width-text-group">
             <div class="m-full-width-text">
-              <ul>
-                <li>Gray Well</li>
-                <li>Well's content</li>
-              </ul>
```

Comment:
I included in the docs because there's confusion as to where the bulleted lists are.